src/internal/connector/graph_connector_test.go

```diff
@@ -133,8 +134,10 @@ func (suite *GraphConnectorUnitSuite) TestUnionSiteIDsAndWebURLs() {
 
 type GraphConnectorIntegrationSuite struct {
 	suite.Suite
-	connector *GraphConnector
-	user      string
+	connector   *GraphConnector
+	user        string
+	acct        account.Account
+	credentials account.M365Config
```

Comment:
This field has become unused. Need follow up Issue to remove it from the struct.